.gitignore

```diff
@@ -6,8 +6,13 @@
 pyenv/
 pyenv2/
 
+# tool-generated folders
+prof/
+IATI-Data-Snapshot/
```

Comment:
Not sure we should be adding a tool folder to the `.gitignore`. This sounds more like a usage of the library, rather than part of the library itself.

Reply:
https://github.com/IATI/iati.core/pull/142/commits/0881dc3c0ae70477752e7be527c37d7539332907